model/faas/spans.yaml

```diff
@@ -26,6 +27,9 @@ groups:
           call to invoke the lambda, which is often HTTP).
       - ref: faas.invocation_id
       - ref: cloud.resource_id
+        deprecated:
```

Comment:
I am strongly against deprecating cloud.resource_id. There should be no harm in setting it in addition to faas.function.id if on a cloud, and generic cloud backend logic not aware of faas.function.id would break and require extra code after this change.

Besides, we originally had faas.id but explicitly removed it in favor of the more generic cloud.resource_id. Reverting such an explicit decision IMHO requires a stronger motivation than this issue + PR currently provides (or at least documents). See https://github.com/open-telemetry/opentelemetry-specification/pull/3188

For example, it is not clear why one can't set cloud.resource_id even outside a classic cloud, e.g. by adding a special cloud.provider (like "on-premise").

(EDIT: moved from md to yaml)

Reply:
The cloud attributes are not aws specific.